<?xml version="1.0"?>
<launch>

    <include file="$(find probot_gazebo)/launch/probot_anno_with_gripper/probot_anno_with_gripper_bringup_moveit.launch" />

    <node pkg="probot_grasping" name="graspingDemo" type="graspingDemo" output="screen">
        <param name="table_length"          value="0.3"/>
        <param name="table_width"         value="0.3"/>
        <param name="initpos_x"            value="0"/>
        <param name="initpos_y"            value="0"/>
        <param name="initpos_z"            value="0.28"/>
    </node>

</launch>
